BlogTemplate/Models/ExcerptGenerator.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BlogTemplate.Models
+{
+    public class ExcerptGenerator
+    {
+        private BlogDataStore _dataStore;
+
+        public ExcerptGenerator(BlogDataStore dataStore)
+        {
+            _dataStore = dataStore;
+        }
+
+        public string CreateExcerpt(string body)
+        {
+            string excerpt;
+            if (body.Length > 140)
```

Comment:
instead of using hardcoded numbers, you could pass in a desired length parameter.